static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -307,17 +359,11 @@ function WidgetViewerModal(props: Props) {
   );
 
   let columnOrder = decodeColumnOrder(
-    tableWidget.queries[0].fields?.map(field => ({
+    fields.map(field => ({
       field,
     })) ?? []
```

Comment:
I think with us using `fields.map` we can drop the `?? []` since it'll always be an array

Reply:
👍 good observation!